test/libsolidity/util/TestFileParser.cpp

```diff
@@ -674,6 +694,19 @@ string TestFileParser::Scanner::scanHexNumber()
 	return number;
 }
 
+string TestFileParser::Scanner::scanReactionString()
+{
+	string str;
+	advance();
+
+	while (current() != '.')
```

Comment:
Not sure about this. I chose this "terminator" because the reactions would then look like sentences.
```
// smoke.action ->
// - Hello this is a reaction.
// - Event "A(address,bytes32,uint256)" contained 0x1212121212121212121212121212120000000012, 0x1234, 0x17.
```
Of course this also means that we cannot use a '.' within a reaction.

Reply:
maybe ';' is better

Reply:
Wouldn't newline be the natural terminator?

Reply:
Newlines are definitely the most natural terminator. I think the newlines got already removed here, will check this.